components/02-elements/form/input/input.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="input">
+<div class="input {{#if modifier}}input--{{modifier}}{{/if}}">
```

Comment:
We are not following `modifier` approach anymore, b/c ins't flexible enough.
Add variable `class` and render it, without any conditions.